crates/nostr/src/lib.rs

```diff
@@ -19,6 +19,8 @@ pub use self::key::Keys;
 pub use self::message::{ClientMessage, RelayMessage, SubscriptionFilter};
 pub use self::metadata::Metadata;
 
+pub use secp256k1::hashes::sha256::Hash as Sha256Hash;
```

Comment:
And this, should go above the `mod` declaration

Like this:
```rust
#[macro_use]
extern crate serde;

pub use secp256k1::hashes::sha256::Hash as Sha256Hash;

pub mod contact;
pub mod event;
```

Again, sorry for nitpicking